python/ambassador/ir/irambassador.py

```diff
@@ -113,6 +114,7 @@ def __init__(self, ir: 'IR', aconf: Config,
             use_ambassador_namespace_for_service_resolution=False,
             server_name="envoy",
             debug_mode=False,
+            preserve_external_request_id=False
```

Comment:
I think that this logic is correct, but there's a comma missing after the `False` which makes me suspect that this'll blow up. 🙂 How about fix that and we can try to get this landed?

Reply:
opps, thanks for pointing it out @kflynn. The changes have been done. Thanks